data-prepper-plugins/key-value-processor/src/test/java/org/opensearch/dataprepper/plugins/processor/keyvalue/KeyValueProcessorTests.java

```diff
@@ -371,6 +372,36 @@ void testDeleteValueAndKeyRegexKvProcessor() {
         assertThatKeyEquals(parsed_message, "key2", "value2");
     }
 
+    @Test
```

Comment:
Not required -- Since the testing logics are similar among those 3 test cases, I would encourage using @ParameterizedTest